### PR TITLE
refactor(test): add some unit tests for dagent

### PR DESCRIPTION
**Pull request by Fall1ngStar**
Add unit tests in `dagent` package to increase code coverage (#241)

### CODE REVIEW

**Comment by Fall1ngStar** [comment]
I've added unit tests to as much pure functions as I could, if you have any tips on how to add tests for functions that interact with other components, I'll be glad to add more tests

File: agent/pkg/dagent/caps/parser_test.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import (
+	"encoding/json"
+	v1 "github.com/dyrector-io/dyrectorio/agent/api/v1"
+	"github.com/dyrector-io/dyrectorio/agent/pkg/builder/container"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestParseLabelsIntoContainerConfig_UnrelatedValue(t *testing.T) {
+	a := map[string]string{
```
Comment (nandor-magyar) [inline agent/pkg/dagent/caps/parser_test.go:15]: Please use more meaningful names, like `a: labels`, `b: config`.

File: agent/pkg/dagent/utils/import_container_test.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	v1 "github.com/dyrector-io/dyrectorio/agent/api/v1"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCheckIfTargetVolumeIsThere(t *testing.T) {
+	ic := &v1.ImportContainer{
+		Volume: "foo",
+	}
+	ml := []mount.Mount{
```
Comment (nandor-magyar) [inline agent/pkg/dagent/utils/import_container_test.go:17]: ml? More verbose name please.

**Review by nandor-magyar — CHANGES_REQUESTED** [review]

**Review by polaroi8d — APPROVED** [review]
LGTM (informal) thanks anyway good job. Could you please sign your commits?

**Comment by Fall1ngStar** [comment]
Will do !

**Review by nandor-magyar — APPROVED** [review]
:+1:

**Pull request merged**